relentless/potential/pair.py

```diff
@@ -589,9 +589,123 @@ class Depletion(PairPotential):
         List of types (A type must be a str).
 
     """
+
+    class LowBound(core.DependentVariable):
```

Comment:
Let's remove this class if we are not going to enforce it automatically. The user can use `ArithmeticMean` if they want to do this explicitly.